plugin/perf.go

```diff
@@ -40,6 +40,7 @@ func (pp *PerfPlugin) GetPanelConfig() (*PanelConfig, error) {
 		Panels: []UIPanel{
 			{
 				Includes: []template.HTML{
+					`<script type="text/javascript" src="/static/app/common/PointsDataService.js"></script>`,
```

Comment:
@john-m-liu Is this the correct method to inject a new service?

Reply:
No, inject this service with the other ones here: https://github.com/evergreen-ci/evergreen/pull/2141/files#diff-003f0c3d54351e3d65e6da9257ea0e4fR14